src/pages/billing/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard.test.ts

```diff
@@ -30,7 +39,7 @@ const getStep3AddTerraAsUserText = () => textMatcher('Add terra-billing@terra.bi
 const getStep3ContactBillingAdministrator = () => textMatcher('Contact your billing account administrator and have them add you and terra-billing@terra.bio as a ' +
   "Billing Account User to your organization's billing account.")
 
-const getStep4CreateButton = () => screen.queryByText('Create Terra Billing Project')
+const getStep4CreateButton = () => screen.queryByRole('button', { name: 'create-billing-project' })
```

Comment:
Having this aria-label "create-billing-project" is not friendly-- it means the screen reader reads that instead of the button text. There shouldn't be an aria-label for something that doesn't need it.

Reply:
The testing library documentation encourages retrieving elements  by role when possible. 
What I failed to realize at the time, was that `name` also picks up on the element text (or maybe there was some other reason it wasn't working at the time).

Removing the aria-label, in any case.